tests/platform_tests/test_sfp.py

```diff
@@ -171,10 +141,11 @@ def test_check_sfp_status_and_configure_sfp(duthosts, rand_one_dut_hostname, con
         assert parsed_presence[intf] == "Present", "Interface presence is not 'Present'"
 
     logging.info("Check interface status")
-    mg_facts = duthost.get_extended_minigraph_facts(tbinfo)
-    intf_facts = duthost.interface_facts(up_ports=mg_facts["minigraph_ports"])["ansible_facts"]
-    assert len(intf_facts["ansible_interface_link_down_ports"]) == 0, \
-        "Some interfaces are down: %s" % str(intf_facts["ansible_interface_link_down_ports"])
+    for namespace in duthost.get_frontend_asic_namespace_list():
```

Comment:
Can you use enum_asic_index?

Reply:
I checked this test, there are certain utility scripts used here like sfputil which doesn't take the namespace argument and gives all the interfaces in the DUT. Hence I feel it is easier to consider the DUT as a whole.

For the CLI show commands for multi-asic support, we had enhanced a different script "sfpshow" to accept the namespace parameter. I will try if I can use sfpshow instead of sfputil here in this test.

Reply:
Updated the tests to use the asic_index. There is a TODO statement I have added, which I will take care of removing when minigraph_facts support namespaces.